sdk/core/azure-core/src/main/java/com/azure/core/util/Header.java

```diff
@@ -25,7 +35,21 @@
     public Header(String name, String value) {
         Objects.requireNonNull(name, "'name' cannot be null.");
         this.name = name;
-        this.value = value;
+        this.values = new LinkedList<>();
+        this.values.add(value);
+    }
+
+    public Header(String name, String... values) {
+        Objects.requireNonNull(name, "'name' cannot be null.");
+        this.name = name;
+        this.values = new LinkedList<>();
+        this.values.addAll(Arrays.asList(values));
```

Comment:
It would be better to iterate the values ourselves as `Arrays.asList` will box the values into an internal type then unbox it back to a newly cloned array

Reply:
Funnily enough this is what I initially had, but IntelliJ warned to change it to above so I let it. Reverted it now.